play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -758,6 +759,69 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 		}
 		return null;
 	}
+
+	renderAfterBattleControls() {
+		const room = this.props.room;
+		return <div class="controls">
+			<p>
+				<span style="float: right"><a
+					onClick={this.handleDownloadReplay}
+					href={`//${Config.routes.replays}/`}
+					class="button replayDownloadButton"
+				>
+					<i class="fa fa-download"></i> Download replay</a>
+				<br />
+				<br />
+				<button class="button" name="cmd" value="/savereplay">
```

Comment:
```suggestion
				<button class="button" data-cmd="/savereplay">
```
This is the new way to make command buttons.